shell/common/shell_unittests.cc

```diff
@@ -14,6 +14,7 @@
 #include "flutter/shell/common/rasterizer.h"
 #include "flutter/shell/common/shell.h"
 #include "flutter/shell/common/thread_host.h"
+#include "flutter/shell/gpu/gpu_surface_software.h"
```

Comment:
The following is a general comment on naming that does not need to be addressed in this PR:

The name of "gpu_surface_software" seems a little strange: if it's software, it's not a gpu surface...

This issue exists in many places such as "gpu thread" which may or may not involve gpu. Maybe at some point, we should substitute all "gpu" with "render" so they become "render_surface_software" and "render thread".

Reply:
Agreed. The support for various backends using different client rendering APIs was added much later and these naming patterns are an unfortunate relic of the old way of doing things. These APIs are not public however and so should be straightforward to rename to make more idiomatic.